bin/cylc

```diff
@@ -524,6 +526,8 @@ if len(args) == 1:
         sys.exit(0)
     if (args[0] == '-v' or args[0] == '--version'):
         # cylc -v,--version
+        # is now an alias for "cylc version"
```

Comment:
Sorry - I now understand what you mean!!  I forgot to actually implement the aliasing to the new command.  So my proposal above ("for continuity...") is already in effect; I'll just remove the erroneous comment...